dipy/denoise/tests/test_lpca.py

```diff
@@ -212,9 +213,14 @@ def test_phantom():
     assert_(rmse_den < rmse_noisy)
     assert_(rmse_den_wrc < rmse_noisy_wrc)
 
+    # Check if the results of different PCA methods (eig, svd) are similar
+    DWI_den_svd = localpca(DWI, sigma, pca_method='svd', patch_radius=3)
+    assert_array_almost_equal(DWI_den, DWI_den_svd)
+
     # Try this with a sigma volume, instead of a scalar
     sigma_vol = sigma * np.ones(DWI.shape[:-1])
-    DWI_den = localpca(DWI, sigma_vol, patch_radius=3)
+    mask = np.ones_like(DWI, dtype=bool)[..., 0]
+    DWI_den = localpca(DWI, sigma_vol, mask, patch_radius=3)
```

Comment:
I just corrected it with applying the mask on the original data as well when it comes to calculating rmse values.